src/containers/SceaHomepageSpeciesContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import withFetchLoader from './FetchLoader'
+import cardPropTypes from '../cards/cardPropTypes'
+import SpeciesCard from '../cards/SpeciesCard'
+
+const CardContainer = styled.div`
+  border-radius: 8px;
+  :hover {
+    background: AliceBlue;
+  }
+`
+
+const SceaHomepageSpeciesContainer = ({cards}) =>
```

Comment:
Is it a typo here, `Scea` not `Scxa`?

Reply:
So, history time!
`gxa` was the moniker given to Expression Atlas, as the original project name was “Gene Expression Atlas”, and the “x” was used instead of “e” to give the URL more personality and make it easier to remember. However, nowadays, we prefer to use EA when using an acronym to refer to Expression Atlas.
`scxa` was the internal name of Single Cell Expression Atlas to follow the pattern of `gxa`, but again, we use SCEA more often.

In summary: any way is right, but consider `gxa` and `scxa` historical curiosities.